Application: Added Suppliers

MuhammadAbdullah191 opened the pull request:
-> Added Supplier Model
-> Added Supplier Controller
-> Added Admin Ability to create, edit and delete Suppliers

MuhammadAbdullah191 committed:
Application: Added Suppliers
-> Added Supplier Model
-> Added Supplier Controller
-> Added Admin Ability to create, edit and delete Suppliers

husnainrazadevsinc commented on app/controllers/suppliers_controller.rb:
```diff
@@ -0,0 +1,57 @@
+  private
+  
+  def set_supplier
+    @supplier = Supplier.find(params[:id])
```
please use find_by instead of find especially if you're not rescuing an exception

husnainrazadevsinc reviewed (commented):
All good. One change is suggested. Please check
Also, please break your code pieces into several commits in future

MuhammadAbdullah191 committed:
Feedback Addressed:
-> Added Dafault value to suppliers model
-> Added Dafault value to set_supplier function
-> Added Failure check to supplier delete action
